tests/5.1/target/test_target_declare_indirect.c

```diff
@@ -0,0 +1,61 @@
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+typedef int(*funcptr)();
+
+int fun1(){
+	return 5;
+}
+
+int fun2(){
+	return 10;
+}
+
+int fun3(){
+	return 15;
+}
+#pragma omp begin declare enter(fun1, fun2, fun3) indirect
```

Comment:
```suggestion
#pragma omp declare target enter(fun1, fun2, fun3) indirect
```
(or 'begin/end declare target' around the functions but then without 'enter' clause)